mandatory/2-function-creation.js

```diff
@@ -15,26 +22,54 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200){
+  return totalPrice * 0.9;
+} else {
+  return totalPrice * 0.95;}
+
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let i = 1;
```

Comment:
it is always suggested to use more describing names, "i" seams to short and can be confusing